src/commands/cmd_zset.cc

```diff
@@ -745,6 +822,7 @@ REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandZAdd>("zadd", -4, "write", 1, 1, 1),
                         MakeCmdAttr<CommandZLexCount>("zlexcount", 4, "read-only", 1, 1, 1),
                         MakeCmdAttr<CommandZPopMax>("zpopmax", -2, "write", 1, 1, 1),
                         MakeCmdAttr<CommandZPopMin>("zpopmin", -2, "write", 1, 1, 1),
+                        MakeCmdAttr<CommandZMPop>("zmpop", -4, "write", 1, 1, 1),
```

Comment:
```suggestion
                        MakeCmdAttr<CommandZMPop>("zmpop", -4, "write", 2, 2, 1),
```

You can refer to https://github.com/apache/incubator-kvrocks/pull/1444#discussion_r1201874256 for a more concrete solution.

Reply:
To my understanding, these arguments indicate keys position in the command? So maybe `2, -1, 1` is better?

Reply:
Not precise. You should consider the last position.

Reply:
> Not precise. You should consider where the last position is. `-1` means the last one and it could be a `count`.
> 
> For example, the `ZMPOP 2 myzset myzset2 MIN COUNT 10` command should return the range {2,3,1} because we want to watch `myzset`, `myzset2` in redis if possible. You can see https://github.com/apache/incubator-kvrocks/pull/1444/files#diff-16b6c7fdec7b9f742f389c178e5956068721e86e5683e2d52a88855ad3efeb21R334 to view the `Range` function for syntax `SINTERCARD numkeys key [key ...] [LIMIT limit]`.

So I need to implement an extra function to determine the range right?